frontend/src/scenes/authentication/signup/signupForm/test/SignupForm.tsx

```diff
@@ -2,7 +2,7 @@ import { useEffect, useState } from 'react'
 import { Link } from 'lib/components/Link'
 import { useActions, useValues } from 'kea'
 import { preflightLogic } from 'scenes/PreflightCheck/preflightLogic'
-import { signupLogic } from './signupLogic'
+import { signupLogicTest } from './signupLogicTest'
```

Comment:
This is quite similar to how we name our actual logic tests, ex: `signupLogic.test.ts` perhaps something like `signupTestLogic` is slightly less confusing? Along with `signupControlLogic`, since we're making the naming changes here anyway? 😃

Reply:
Good call, made the changes 👍